dist/hardhat.config.js

```diff
@@ -119,7 +119,7 @@ const config = {
     solidity: {
```

Comment:
do we need the dist folder, or can we include in `.gitignore`?

Reply:
Yessir can be ignored

Reply:
removed and ignored in 5c05159